ligo/stablecoin/spec.ligo

```diff
@@ -240,6 +242,27 @@ end
 
 type transfer_descriptor_param is michelson_pair_right_comb(transfer_descriptor_param_)
 
+(*
+ * SafeList
+ *)
+
+type safelist_transfer_item is record
+  from_ : address
+; to_ : address
+end
+
+type safelist_assert_transfers_param is list(safelist_transfer_item)
```

Comment:
These types below seem to be unused

Reply:
Checked again, since it's a part of our specification then I believe we should leave them but at least clarify that they're unused

Reply:
> Should we typecheck the safe list contract when it is being set?

According to the specification we should. But apparently we don't do it, do we? From what I see, currently `set_safelist` only checks that the sender is owner.

Reply:
Also this type doesn't match the one from the specification:
> `assertTransfers(list(pair (address :from) (address :to))) `

You have a list where each item has a list, but in specification we have `assertTransfers` that takes a list of pairs.

I suppose you have changed it due to recent FA2 changes. I am not sure whether we should update our code or safelist interface should be updated instead. I have brought it up here: https://gitlab.com/tzip/tzip/-/merge_requests/61/diffs#note_351166098

I think we can merge your version for now until we have a definite decision regarding `assertTransfers`. But then please update specification and create a new issue about that.

Reply:
> update specification

I think you can simply add a TODO there for now btw.